### ISSUE TITLE
Please provide basic config files

### ISSUE THREAD

**Issue by nussbaumer** [wizbright/waybox#30]
Could You please provide some basic config files (rc.xml, menu.xml) to start with, and to see what is already implemented?

I only get an empty window without any menus/function, when loading existing openbox configs.
(waybox.git, archlinux/AUR)

Thanks for your work on this promising projekt!

**Comment by keithbowes** [comment]
> Could You please provide some basic config files (rc.xml, menu.xml) to start with, and to see what is already implemented?

Yeah, that's planned for 0.2.0.  Menus still haven't been implemented, but I'm considering going with a high-level approach (using a PyGObject script to make the menus and pipes to communicate between the main program and the menu script).  That will come later.

Right now, you can use keybindings from your Openbox config (mouse bindings will come later).  Most things should just work.  Waybox does use xkbcommon, where Openbox used Xlib, so some keysyms might be different, but I haven't encountered much of a problem in that regard (other than that I can use A-Tab to cycle through windows, but I had to change A-S-Tab to A-S-ISO_Left_Tab to reverse cycle).

> I only get an empty window without any menus/function, when loading existing openbox configs. (waybox.git, archlinux/AUR)

If you don't have an application launcher bound to some key combination (e.g. A-F2), you can use the Openbox-compatible `--startup` command-line argument.  In 0.2.0, my plan is to make it by default use ~/.config/waybox/autostart or /etc/xdg/waybox/autostart (similarly to what Openbox does), but you can currently start any program that way.

Here's a little screenshot of Waybox running Firefox and [Waybar](https://github.com/Alexays/Waybar):
![screenshot](https://user-images.githubusercontent.com/1697807/153942248-2868b8f9-373d-4710-9e6c-928c32854fac.png)

**Comment by q5sys** [comment]
@keithbowes Really glad you're back at this.  I'm the Developer of the Lumina Desktop and I've not been looking forward to dealing with Wayland.  What you're working on looks like it might be the answer to that problem, so I'm really hopefully for what you can pull off.
